share/css/dnsviz.css

```diff
@@ -12,7 +12,7 @@
 .dnsviz-secure { background-color: #8ffeff; }
 .dnsviz-insecure { background-color: #b7b7b7; }
 .dnsviz-bogus { background-color: #f17b7b; }
-.dnsviz-non-existent { background-color: #d0d0d0; }
+.dnsviz-nonexistent { background-color: #d0d0d0; }
```

Comment:
similarly, it's possible this is an api break if someone is reskinning the system...

Reply:
I like the idea, but this is more than just re-skinning.  DNSViz generates the classes dynamically, and I need to see where else this is being used...

Reply:
Let me check back to see what else this affects before changing.  It might be nothing - I just don't know without further review.